single_tech_samples/fabric/feature_engineering_on_fabric/README.md

```diff
@@ -379,6 +400,40 @@ If the pipeline executes successfully, you can verify the features registered in
 
   ![feature_lineage](./images/managed_feature_store.gif)
 
+### Exploratory data analysis (EDA)
```

Comment:
Few comments:
- Suggest moving it to as an H2 header after "Model training and inferencing" as below:

## Model training and inferencing

## Additional (optional) Features

### Exploratory data analysis (no need to put the short form 'EDA' here)

### Data Validations